src/context.ts

```diff
@@ -78,6 +78,9 @@ export async function getAppendArgs(inputs: Inputs, node: Node, toolkit: Toolkit
   const args: Array<string> = ['create', '--name', inputs.name, '--append'];
   if (node.name) {
     args.push('--node', node.name);
+  } else if (inputs.driver == 'kubernetes') {
+    // TODO: make this conditional on buildx version when https://github.com/docker/buildx/pull/1673 is merged
```

Comment:
:eyes:

Reply:
When a new release of buildx will be done I will add a new cond on buildx version to skip this. Keeping the PR in draft in the meantime.